pkg/apis/helm/v1beta1/generic_hash.go

```diff
@@ -52,6 +52,10 @@ func cleanUpMapValue(v interface{}) interface{} {
 	case float64:
 		return v
 	default:
+		// Keep null values as nil to avoid type mismatches
+		if v == nil {
+			return nil
+		}
```

Comment:
would it make sense to have this above the switch?

Reply:
Yeah, probably. Moved it, thank you!